src/overlays/ovl_map_mark_data/z_map_mark_data.c

```diff
@@ -1,6 +1,6 @@
 #include "global.h"
 
-MapMarkData sMapMarkDekuTree[] = {
+static MapMarkData sMapMarkDekuTree[] = {
```

Comment:
can you update https://github.com/zeldaret/oot/blob/master/tools/overlayhelpers/mapmark.py as well